src/test/java/com/sampleweb/journey/CdVinylTests.java

```diff
@@ -0,0 +1,112 @@
+package com.sampleweb.journey;
+
+import com.sampleweb.BaseTest;
+import com.sampleweb.framework.PageNavigator;
+import com.sampleweb.pages.CdVinylPage;
+
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class CdVinylTests  extends BaseTest {
+
+    private PageNavigator navigator = new PageNavigator();
+    private CdVinylPage cdVinylPage;
+
+    @BeforeMethod
+    public void setUp() throws Exception {
+        cdVinylPage = navigator.navigateToPage(driver, CdVinylPage.PATH, CdVinylPage.class);
+    }
+
+    @Test(groups = "regression")
+    public void navigateToPage() {
+        //CDs & Vinyl dropdown
+        String dropDownPath = "//select[@class=\"nav-search-dropdown searchSelect\"]/option[@value=\"search-alias=popular\"]";
+
+        driver.findElementByXPath(dropDownPath).click();
+
+        cdVinylPage.setSearchCriteria("").clickSubmitButton();
+
+        assertThat(cdVinylPage.isLoaded(), is(true));
+    }
+
+
+    @Test(groups = "regression")
+    public void titleTest() {
+        Assert.assertTrue( driver.getTitle().contains(cdVinylPage.TITLE) );
+    }
+
+    @Test(groups = "regression")
+    public void checkGlobalStoreSection() {
+        WebElement storeElement = driver.findElementByXPath(cdVinylPage.GLOBAL_STORE_CHECKBOX_PATH);
+
+        //if query can't find an object, fail test
+        if(storeElement == null) {
+            Assert.fail();
+        }
+    }
+
+    @Test(groups = "regression")
+    public void checkAverageCustomerReview() {
+        WebElement reviewElementTitle = driver.findElementByXPath("//h4[text() = \"Avg. Customer Review\"]");
+
+        if(reviewElementTitle == null) {
+            Assert.fail();
+        }
+
+        //Only stars are in a unordered list with icons
+        WebElement starElementList = driver.findElementByXPath("//ul/div/li/span/a/i");
+        if(starElementList == null) {
+            Assert.fail();
+        }
+
+    }
+
+    //Its grouped under music so title is the 4th element
+    @Test(groups = "regression")
+    public void fourthLinkHeaderIsCategory() {
+        String elementText = driver.findElementByXPath(cdVinylPage.CD_VINYL_LINK_PATH).getText();
+        Assert.assertEquals(elementText, cdVinylPage.TITLE);
+    }
```

Comment:
Would it be more clear if you were asserting true like in the examples? Have a function in the page which return boolean?

Reply:
I think `assertEquals` is fine but I would add a static import for it